ml/ml.cc

```diff
@@ -1498,9 +1554,12 @@ bool ml_dimension_is_anomalous(RRDDIM *rd, time_t curr_time, double value, bool
 }
 
 static void ml_flush_pending_models(ml_training_thread_t *training_thread) {
-    int rc = db_execute(db, "BEGIN TRANSACTION;");
     int op_no = 1;
 
+    // begin transaction
+    int rc = db_execute(db, "BEGIN TRANSACTION;");
```

Comment:
Transaction is not needed (a single command is wrapped in a transaction by itself)